rfc/text/advanced/ap_27_pubsub_event_history.md

```diff
@@ -1,58 +1,132 @@
 ## Event History {#pubsub-event-history}
 
-Instead of complex QoS for message delivery, a *Broker* may provide *message history*. A *Subscriber* is responsible to handle overlaps (duplicates) when it wants "exactly-once" message processing across restarts.
-
-The *Broker* may allow for configuration on a per-topic basis.
+Instead of complex QoS for message delivery, a *Broker* may provide *message history*. A *Subscriber* is 
+responsible to handle overlaps (duplicates) when it wants "exactly-once" message processing across restarts.
 
 The event history may be transient or persistent message history (surviving *Broker* restarts).
 
-A *Broker* that implements *event history* must (also) announce role `HELLO.roles.callee`, indicate `HELLO.roles.broker.history == 1` and provide the following (builtin) procedures.
+The *Broker* implementation may allow for configuration on a per-topic basis.
```

Comment:
"... allow for configuration of event history on a per-topic or on a per-topic pattern basis. Such configuration could enable/disable the feature, event history storage location and parameters such as compression, or the event history data retention policy."

Reply:
👍🏻  Added!